.github/workflows/integration-tests.yml

```
@@ -0,0 +1,274 @@
+          sudo apt-get update
+          sudo apt-get install -y git
+          git --version
+      
+      - name: Set up git for integration tests
+        run: |
+          git config --global user.name "Integration Tests"
+          git config --global user.email "integration-tests@vibe-ensemble.local"
+          git config --global init.defaultBranch main
+      
+      - name: Cache cargo dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)